app/scripts/lib/country-telephone-info.js

```diff
@@ -91,7 +91,7 @@ define(function(require, exports, module) {
       normalize: ensurePrefix('+61'),
       pattern: /^(?:\+61\d{9}|\d{10})$/,
       prefix: '+61',
-      rolloutRate: 0 // being soft launched. Testers will need to open `/sms?service=sync&country=AU`
+      rolloutRate: 0.5 // being soft launched. Testers will need to open `/sms?service=sync&country=AU`
```

Comment:
@rfk could you please try this when you get a sec ? 🇦🇺

Reply:
I successfully sent myself an SMS via https://accounts.stage.mozaws.net/sms?service=sync&country=AU, the SMS-sending part seems to be working as intended.

I didn't get the cool magic-signin-code-deeplink experience though; @vbudhram should I have?

Reply:
>I didn't get the cool magic-signin-code-deeplink experience though

I wonder if this is because I had previously installed Firefox on this device, which I recall interfering with the deeplink functionality in some cases.

Reply:
> I had previously installed Firefox on this device

Yea, user would not get the deeplink experience in this case. However, you can clear the ID using steps [here](https://github.com/mozilla/fxa/tree/master/features/FxA-54-connect-another-device-phase-3#forget-the-device-with-adjust-1), and then reinstall Firefox which would give you the experience.